test/onnx/test_fx_op_consistency.py

```diff
@@ -147,6 +147,11 @@ def skip_torchlib_forward_compatibility(
         dtypes=(torch.float16,),
         reason="fixme: Assertion error: result mismatch and type error",
     ),
+    skip(
+        "_batch_norm_with_update",
```

Comment:
I guess if onnx export is already broken for the old op we don't need to immediately fix it for the new one 🤔